tools/utils/Utils/AppxPackaging/AppxBundleMetadata.cs

```diff
@@ -122,13 +130,25 @@ private void PopulateChildAppxPackages()
                 {
                     this.ExternalAppxPackages.Add(new ExternalPackageReference(
                         this,
-                        subPackageInfo.GetPackageId().GetPackageFullName(),
-                        subPackageInfo.GetFileName(),
+                        subPackageId.GetPackageFullName(),
+                        filePath,
                         isOptional: false));
                 }
                 else
                 {
-                    this.InternalAppxPackagesRelativePaths.Add(subPackageInfo.GetFileName());
+                    var childPackageMetadata = new ChildPackageMetadata(
+                        filePath,
+                        subPackageInfo.GetPackageType(),
+                        subPackageInfo.GetSize(),
+                        subPackageId.GetPackageFullName(),

```

Comment:
Nit: Maybe it would be good to follow the same order in both constructors. I.e. parent bundle, package full name, relative file path, and then the others.

Reply:
Done, also added more properties that are on ExternalPackageREference